### DISCUSSION TITLE
[low-prio] Ability for friends to see all messages (they have received)

### DISCUSSION THREAD

**Discussion by michael** [PostOwl/postowl#16] [Ideas]
Imagine I receive 20 private letters from you, then the only way to access them is to find the emails and click the secret links (or bookmark them). It somehow would be nice if I could "log in" to your site as a friend, and then see all letters that you shared with me.

To make this work, I could go to keybits.net and enter my email address (the one that is in your friend list) and then I receive a one-time password. Then I have a session on your website which allows me to see all posts that I have access to. :)

**Comment by rehabHassanein** [comment]
Or maybe allow people to "follow"/"subscribe" to each other, you can then see both the public posts as well any private posts that were shared with you.

___________________________________________________________

edit: just realized after typing this that these pages are distributed and don't share a database :thinking:

**Reply by rehabHassanein** [reply]
>I receive a one-time password.

This would mean someone would need to re-generate these passwords when they expire, how about allowing users to signup to the website as a viewer?
